src/wellcode_cli/jira/models/metrics.py

```
@@ -0,0 +1,519 @@
+                priority = fields.get("priority", {})
+                if priority:
+                    priority_name = priority.get("name", "Unknown")
+                    self.by_priority[priority_name].append(cycle_time)
+
+                # Track by issue type
+                issue_type = fields.get("issuetype", {})
+                if issue_type:
+                    type_name = issue_type.get("name", "Unknown")
+                    self.by_issue_type[type_name].append(cycle_time)
+
+        except (ValueError, TypeError) as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)